ebcDockerBuilderRCO.jenkinsfile

```diff
@@ -85,6 +85,7 @@ timestamps {
 def gitCloneAndStash() {
   dir('runtime-component-operator') {
       git branch: RELEASE_TARGET, url: "git@github.com:${scriptOrg}/runtime-component-operator.git"
```

Comment:
You also need to change this line to be:
git branch: "master", url: "git@github.com:${scriptOrg}/runtime-component-operator.git"